addons/binding/org.openhab.binding.wso2iots/OSGI-INF/.gitignore

```diff
@@ -0,0 +1 @@
+*.xml
```

Comment:
I'm not sure whether bindings should contain their own `.gitignore` files

Reply:
For this directory it should 😔 At least for now